Dockerfile

```diff
@@ -1,5 +1,5 @@
 # base image
-FROM pelias/libpostal_baseimage
+FROM pelias/baseimage
```

Comment:
If we want to continue to be able to load libpostal directly, we need to continue to use the `libpostal_baseimage`

For the record I believe the reduced complexity of having only a single way to interact with libpostal (via the libpostal service), is preferable to any performance reduction we might see during builds with having to go over the network.